MeshLib/MeshInformation.cpp

```diff
@@ -19,15 +19,16 @@
 namespace MeshLib
 {
 
-const std::pair<unsigned, unsigned> MeshInformation::getValueBounds(const MeshLib::Mesh &mesh)
+const std::pair<int, int> MeshInformation::getValueBounds(const MeshLib::Mesh &mesh)
 {
-	const std::vector<MeshLib::Element*> &elements (mesh.getElements());
-	const auto minmax = std::minmax_element(elements.cbegin(), elements.cend(),
-        [](MeshLib::Element const*const a, MeshLib::Element const*const b)
-            {
-                return a->getValue() < b->getValue();
-        });
-	return std::make_pair<unsigned, unsigned>((*minmax.first)->getValue(), (*minmax.second)->getValue());
+	boost::optional<MeshLib::PropertyVector<int> const&> materialIds
+		= mesh.getProperties().getPropertyVector<int>("MaterialIDs");
+	if (materialIds) {
+		auto mat_bounds = std::minmax_element(materialIds->cbegin(), materialIds->cend());
+		return std::make_pair(*(mat_bounds.first), *(mat_bounds.second));
+	}
+	else
+		return std::make_pair(-1,-1);
```

Comment:
(-1, -1) pair is not distinguishable from valid bounds of materials = {-1, -1, ..., -1};